packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -36,7 +36,7 @@ export default function({ types: t }) {
       VALUE: value ? value : t.identifier("undefined"),
```

Comment:
Probably better to use `t.unaryExpression('void', t.numericLiteral(0))` to avoid any potential shadowing issues